s3migrate/README.md

```diff
@@ -0,0 +1,154 @@
+# Migrating from mogile to S3
+
+This folder contains code for migrating from Mogile to S3.
+
+In broad strokes, the migration from mogile to S3 consists of the following steps:
+
+1. Build and configure the migration.
+1. Sync data from mogile to an AWS bucket
+1. Remove or fix bad data.
+1. Run a large-scale migration in AWS lambda.
+1. Stop ingesting new data into contentrepo.
+1. Run a large-scale migration in AWS lambda.
+1. Verify migration.
+1. Update contentrepo configuration.
+1. Update data in contentrepo and rhino databases to point to the new, S3 buckets.
+1. Done!
+1. Test
+1. Re-enable ingest.
```

Comment:
Is it implied that we're cutting over from the SOMA Journals stack to the AWS Journals stack in this process? I'm kinda unclear on how related this process is to the Journals migration.

Reply:
I'm not sure, but this process should work whether or not we are moving to AWS journals stack at the same time.

Reply:
agreed, yeah, I think it'll work either way. Do you know when we'll make that decision? (to migrate crepo to s3 before or during the journals stack migration to AWS?)

Reply:
I'm mainly curious because some of the details that are currently elided by these instructions will change, for instance _where_ we re-enable ingest (SOMA or AWS)

Reply:
I don't know when we'll make the decision. I suppose it partially depends on how far along we are with the journal stack on AWS. Perhaps @sgatchell knows moer.

Reply:
I don't know about deployment plans, but I'd prefer to the S3 migration done before moving the journals apps to AWS. 

The salt changes are nearly complete, with pretty much just wombat remaining.

Reply:
> I don't know about deployment plans, but I'd prefer to have the S3 migration done before moving the journals apps to AWS.

Sounds good to me -- that seems like we're dealing with less complexity at once.

Reply:
>>> Sounds good to me -- that seems like we're dealing with less complexity at once.

Same